mappings/net/minecraft/client/gui/screen/multiplayer/SocialInteractionsScreen.mapping

```diff
@@ -40,6 +40,10 @@ CLASS net/minecraft/unmapped/C_brwvrttn net/minecraft/client/gui/screen/multipla
 		ARG 1 button
 	METHOD m_eicljhoo setOnRendered (Ljava/lang/Runnable;)V
 		ARG 1 onRendered
+	METHOD m_klplqoox (Z)V
+		ARG 1 yesButton
```

Comment:
`yesButton` seems a bit off, `acceptButton` or `confirmButton` imo

Reply:
changed to acceptButton